docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+### Tree-sitter lands in Atom
+
+<img src="./assets/tree-sitter-lands.png" height="200" />
+
+You may have read our [series of blog posts](https://pulsar-edit.dev/tag/tree-sitter/) from [@savetheclocktower] and [@maurício szabo] that detail the huge amount of work that has been going on to upgrade our Tree-sitter implementation. This all goes back to 2018 when [Max Brunsfeld announced](https://github.blog/2018-10-31-atoms-new-parsing-system/) that Atom 1.32 would be shipping with a brand new parsing system that promised to be better at understanding your code and making it easier to write - Tree-sitter. Like how Atom Shell was created for Atom and was then widely used elsewhere, the same can be said of Tree-sitter, which now sits at the heart of a number of other projects: Helix, Zed, and even Github's own [Linguist](https://github.com/github-linguist/linguist) library.
+
+### GitHub sunsets Atom
+
+<img src="./assets/sunsetting2.png" height="200" />
+
+So, having just mentioned how Microsoft had promised that Atom and VS Code would both see active development and support, it became clear that interest in Atom development was decreasing, new features weren't forthcoming, and issues were being left unresolved. The Atom.io blog page saw its last update in 2019 with the announcement of Atom 1.39, and most releases after that featured fixes and updates, but little in the way of exciting new features. The old Atom forum was shut down (but it is still browsable via the [Wayback Machine](https://web.archive.org/web/20210622052601/https://discuss.atom.io/)), and moved to the, then new, [GitHub Discussions](https://github.com/atom/atom/discussions) page, but there was little interaction from the original developers. The original package website was becoming unmaintained and hadn't had the "Featured" or "Trending" packages updated in a long time.
+
+Finally, on the 8th of June, 2022, GitHub made an announcement that Atom would be [sunset](https://github.blog/2022-06-08-sunsetting-atom/) in favour of VS Code and gave 6 months warning before the various services that make up the Atom ecosystem would go offline forever. The software itself and its source code would, of course, still be available via the archived repository, but the website, flight manual, package backend, and Teletype service would no longer be available.
+
+### Atom Community
+
+<img src="./assets/atom-community.png" height="200" />
+
+As soon as the news hit, there was a flurry of activity. Some were looking for direct replacements for Atom; others were looking for active forks or continuations of the project. [Atom Community](https://atom-community.github.io/) was the logical place for these discussions. This was a community that had taken on the maintenance of a bunch of Atom packages (originally for the Abandoned Facebook [Nuclide](https://nuclide.io/) project to turn Atom into an IDE) and had since taken on other packages from the original Atom maintainers.
```

Comment:
```suggestion
As soon as the news hit, there was a flurry of activity. Some were looking for direct replacements for Atom; others were looking for active forks or continuations of the project. [Atom Community](https://atom-community.github.io/) was the logical place for these discussions. This was a community that had taken on the maintenance of a bunch of Atom packages (originally for the abandoned Facebook [Nuclide](https://nuclide.io/) project to turn Atom into an IDE) and had since assumed stewardship of other packages from the original Atom maintainers.
```